flink-runtime/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/SlidingEventTimeWindowsTest.java

```diff
@@ -215,6 +215,15 @@ void testInvalidParameters() {
                                         Duration.ofSeconds(11)))
                 .isInstanceOf(IllegalArgumentException.class)
                 .hasMessageContaining("abs(offset) < slide and size > 0");
+
+        assertThatThrownBy(
+                        () ->
+                                SlidingEventTimeWindows.of(
+                                        Duration.ofMillis(
+                                                SlidingEventTimeWindows.MAX_WINDOW_NUM + 1),
+                                        Duration.ofMillis(1)))
+                .isInstanceOf(IllegalArgumentException.class)
+                .hasMessageContaining("size / slide <= 10000000");
```

Comment:
nit: using SlidingEventTimeWindows.MAX_WINDOW_NUM here as well.

Reply:
Personally, I think it is just for UT. But using `SlidingEventTimeWindows.MAX_WINDOW_NUM` is good to me.